packages/docusaurus-theme-classic/src/theme/DocSidebar/index.js

```diff
@@ -27,6 +27,11 @@ function DocSidebarItem({item, onItemClick, collapsible}) {
     setCollapsed(item.collapsed);
   }
 
+  const handleItemClick = useCallback(e => {
```

Comment:
This uses of `useCallback` is premature optimization IMO.

Reply:
Maybe revert it?